pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/Connection.java

```diff
@@ -34,14 +35,23 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(Connection.class);
   private final PinotClientTransport _transport;
   private final BrokerSelector _brokerSelector;
+  private final Config _config;
 
   Connection(List<String> brokerList, PinotClientTransport transport) {
-    LOGGER.info("Creating connection to broker list {}", brokerList);
-    _brokerSelector = new SimpleBrokerSelector(brokerList);
-    _transport = transport;
+    this(new Config(), new SimpleBrokerSelector(brokerList), transport);
+  }
+
+  Connection(Config config, List<String> brokerList, PinotClientTransport transport) {
```

Comment:
Suggest directly passing in the `Properties`. Don't see much value wrapping it into a custom object. `Properties` is more general and easy to be extended in the future